scripts/query-spec

```diff
@@ -9,37 +29,58 @@ if [ "${VERBOSE:-0}" -ge 2 ] || [ "${DEBUG:-0}" -eq 1 ]; then
     ERR_OUTPUT="/dev/stderr"
 fi
 
-if [ $# -lt 2 ]; then
-    echo "Usage: $0 file.spec field" >&2
+source_dir="$1"
+spec_file="$2"
+query_field="$3"
+dist="$4"
+
+if [ $# -lt 4 ]; then
+    echo "Usage: $0 source_dir file.spec field dist" >&2
     exit 1
 fi
 
-[[ "x$(basename "${1}.in")" == "x.in" ]] && exit 0
+rpm_defines=()
+
+# make it a function and export, as bash doesn't support exporting an array
+set_rpm_defines() {
```

Comment:
This one is only used by `query-builtrpms`, which is not part of this repository.